Ginger/GingerInstallerScript.iss

```diff
@@ -124,7 +124,7 @@ Begin
     
     XMLDocument.async := False;
     XMLDocument.load(strUserProfileXMLFilePath);
-    XMLNode := XMLDocument.selectSingleNode('//Ginger.UserProfile');
+    XMLNode := XMLDocument.selectSingleNode('//GingerRepositoryItem//UserProfile');
     if (XMLNode.getAttribute('UserType') = null) then 
```

Comment:
this will work on 2 below scenarios?
1- i have UserProfile created by Ginger 2.7.0 and i'm installing Ginger 3.0.12
2- i have UserProfile created with Ginger 3.0.12 and then i uninstalled 3.0.12 and trying to install it back

Reply:
I pushed more code to support above requirements,
please review,
Thanks.